packages/valory/skills/twitter_scoring_abci/behaviours.py

```diff
@@ -995,9 +995,9 @@ def get_update_diff(self) -> Dict:
         is_period_changed = today != current_period
         if is_period_changed:
             self.context.logger.info(
-                f"Scoring period has changed from {current_period} to {today}"
+                f"Scoring period has changed from {current_period} to {today}. Resetting user period points..."
             )
-        period_reset_users = set()
+            ceramic_db_copy.reset_period_points()
```

Comment:
All user limits need to be reset after each period, not only the ones that tweeted